lib/seattleflu/db/cli/command/longitudinal.py

```diff
@@ -0,0 +1,278 @@
+"""
+Parse and upload longitudinal data.
+
+Longitudinal childcare data will contain PII (personally identifiable
+information) and unnecessary information that does not need to be stored. This
+process will only pull out specific columns of interest that will then be stored
+in the receiving schema of ID3C.
+"""
+import re
+import click
+import logging
+import numpy as np
+import pandas as pd
+import seattleflu.db as db
+from math import ceil
+from seattleflu.db.session import DatabaseSession
+from seattleflu.db.cli import cli
+from . import LOG, add_metadata, generate_hash, barcode_quality_control
+
+
+@cli.group("longitudinal", help = __doc__)
+def longitudinal():
+    pass
+
+@longitudinal.command("parse")
+@click.argument("baseline_filename",
+    metavar = "<Longitudinal Childcare Data baseline (enrollments) filename>")
+@click.argument("weekly_filename",
+    metavar = "<Longitudinal Childcare Data weekly assessments filename>")
+@click.option("-o", "--output",
+    metavar="<output filename>",
+    help="The filename for the output of missing barcodes")
+
+def parse(baseline_filename, weekly_filename, output):
```

Comment:
I would love to walk through this in person 😮

Reply:
It would be my pleasure :smile: